lib/WP_Auth0_Users.php

```diff
@@ -84,6 +84,8 @@ public static function create_user( $userinfo, $role = null ) {
 			'description'  => $description,
 		];
 
+		$user_data = apply_filters( 'wpa0_user_data', $user_data, $userinfo, $firstname, $lastname );
```

Comment:
The naming convention for hooks isn't 100% consistent but we've been using `auth0_` as the namespace for new ones. Can you update the filter name to `auth0_create_user_data` to add that namespace and make this more explicit?

Also, the `$firstname` and `$lastname` parameters are not necessary since their values are already in the array.

Reply:
Thank you for review 👍 Happy to hear that this PR will be merged 😄 
I just updated all what you pointed. Please check it again.